dbt/adapters/dremio/connections.py

```diff
@@ -258,10 +258,10 @@ def execute(
     def drop_catalog(self, database, schema):
         logger.debug('Dropping schema "{}.{}"', database, schema)
 
-        connection_param = self.get_thread_connection()
-        connection = self.open(connection_param)
+        connection = self.get_thread_connection()
```

Comment:
I don't think we should have connection parameters initialise with the same name as the thread connection. I see that both get_thread_connection() and open() return a Connection type (https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/adapters/base/connections.py), but we should still distinguish between the two. Maybe change the first one to thread_connection?

Reply:
done